account.js

```diff
@@ -1,6 +1,9 @@
 /* global QamineUtils */
 
 "use strict";
+console.log("log");
+
+eval("js here");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [eval can be harmful.](https://app.codacy.com/gh/qamine-test/projectWithIssues/pullRequest?prid=5402448)